person_capture/gui_app.py

```
@@ -8281,6 +8558,117 @@ def _quantile_lut(src_ch: np.ndarray, dst_ch: np.ndarray, mask: np.ndarray) -> O
                 pass
             return 0, ""
 
+    def _wic_yuv444_color_match_ref_max_side(self) -> int:
+        try:
+            default_ref = int(getattr(self.cfg, "hdr_wic_yuv444_color_match_ref_max_side", 960) or 0)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)